richtext/plugins/org.polarsys.kitalpha.richtext.widget.ext/plugin.xml

```diff
@@ -35,14 +35,6 @@ Contributors:
             handler="org.polarsys.kitalpha.richtext.widget.tools.ext.types.DiagramElementLinkHandler"
             type="modelDiagram">
       </LinkHandler>
-      <LinkHandler
```

Comment:
The Absolute and Project Relative paths are tagged as not recommanded in the documentation (usersguide.mediawiki) but the link magement are removed. Is this normal?
Or May be that handler are no more required because the image paths is transformed!!

Reply:
In the rich text editor you have two kind of references to an image
* <img ser="path to image" /> It corresponds to the image displayed -> this is no longer used. That is what is removed because now the selection is based on a Sirius component.
* <a href="hyperlink to file"/>. It corresponds to the hyperlink to a file. This one is still alive

Reply:
Thanks for the inforamtion